osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -256,8 +256,22 @@ public void PostCommand(string text, Channel target = null)
                     JoinChannel(channel);
                     break;
 
+                case "chat":
+                case "query":
+                    if (string.IsNullOrWhiteSpace(content))
+                    {
+                        target.AddNewMessages(new ErrorMessage("Usage: /chat [user]"));
```

Comment:
This is now wrong for the alias

Reply:
Fixed, but this whole method seems pretty rudimentary.